jobbergate-agent/jobbergate_agent/jobbergate/schemas.py

```diff
@@ -94,17 +94,12 @@ class SlurmSubmitResponse(pydantic.BaseModel, extra=pydantic.Extra.ignore):
     job_id: Optional[int]
 
 
-class SlurmSubmittedJobStatus(pydantic.BaseModel, extra=pydantic.Extra.ignore):
+class SlurmJobData(pydantic.BaseModel, extra=pydantic.Extra.ignore):
```

Comment:
Could we remove the prefixes `slurm_job` from the fields? The class already makes clear they detail a slurm job, I don't think we need repetition on the fields

Reply:
Well, the reason that we are capturing it in this way is because the API requires the prefixes on its endpoints. For the API, this makes sense since it is storing them in the job_submissions table. There, having the `slurm_job` prefix makes sense.

Reply:
Partly resolved in commit e2e8554. The fields retain the prefix on the API side, but in the agent code, the prefixes are removed.